epub34/authoring/index.html

```diff
@@ -221,12 +221,27 @@ <h4>Relationship to HTML</h4>
 						the various changes to HTML and the technologies it references to ensure they keep their
 						processes up to date.</p>
 
+					<p>The [[html]] standard defines a single content model with rules for expressing a tag set using
+						either the <a data-cite="html#syntax">HTML syntax</a> (what has always been considered HTML) or
```

Comment:
```suggestion
						either the <a data-cite="html#syntax">HTML syntax</a> (what has always been considered as the default for HTML) or
```

I am not sure about the exact details of the proposed change, but somehow it did not sound good without something there...

Reply:
I've removed these parentheticals as they don't seem relevant. The note for html content documents already explains that xhtml content documents was never xhtml 1.1. This is just doing a similar equating of the syntaxes to the old documents.